l2geth/cmd/geth/usage.go

```diff
@@ -73,7 +73,6 @@ var AppHelpFlagGroups = []flagGroup{
 			utils.Eth1StandardBridgeAddressFlag,
 			utils.Eth1ChainIdFlag,
 			utils.RollupClientHttpFlag,
-			utils.RollupAddressManagerOwnerAddressFlag,
```

Comment:
We should also remove the configuration from start.sh, l2geth/README.md, and ops/scripts/geth.sh.